LICENSE

```diff
@@ -1,4 +1,4 @@
-Copyright (c) 2011,2012,2013,2014,2015,2016,2017 Genome Research Ltd.
+Copyright (c) 2011-2021 Genome Research Ltd.
```

Comment:
Last time I saw it Sanger policy cautioned against date ranges unless they were impossible to avoid (Ie. you didn't know if there had been commits on certain years)